connectors/ssh/src/main/java/org/crsh/ssh/util/KeyPairUtils.java

```diff
@@ -23,7 +23,12 @@
 import java.io.Reader;
 
 /** @author <a href="mailto:tuyennt@exoplatform.com">Tuyen Nguyen The</a> */
-public class KeyPairUtils {
+public final class KeyPairUtils {
+	
+	private KeyPairUtils(){
+		throw new AssertionError("Must not instantiate this class");
```

Comment:
If this is really necessary, perhaps it should throw an IllegalStateException, instead.